Dockerfile

```diff
@@ -76,88 +78,63 @@ RUN wget https://nodejs.org/download/release/v6.14.4/node-v6.14.4-linux-x64.tar.
-MAINTAINER Denys Zhdanov <denis.zhdanov@gmail.com>
+LABEL maintainer="Denys Zhdanov <denis.zhdanov@gmail.com>"
+
+ENV STATSD_INTERFACE udp
 
 # choose a timezone at build-time
 # use `--build-arg CONTAINER_TIMEZONE=Europe/Brussels` in `docker build`
 ARG CONTAINER_TIMEZONE
-ENV DEBIAN_FRONTEND noninteractive
 
 RUN if [ ! -z "${CONTAINER_TIMEZONE}" ]; \
     then ln -sf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && \
     dpkg-reconfigure -f noninteractive tzdata; \
     fi
 
-RUN apt-get update --fix-missing \
-    && apt-get -y upgrade \
-    && apt-get install --yes --no-install-recommends \
-    git \
-    redis \
-    collectd \
-    nginx \
-    python3-pip \
-    python3-ldap \
-    expect \
-    memcached \
-    sqlite3 \
-    libcairo2 \
-    librrd-dev && \
-    apt-get clean && \
-    apt-get autoremove --yes  && \
-    rm -rf /var/lib/apt/lists/*
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && apt-get update --fix-missing \
+ && apt-get -y upgrade \
+ && apt-get install --yes --no-install-recommends \
+      collectd \
+      expect \
+      git \
+      libcairo2 \
+      librrd-dev \
+      memcached \
+      nginx \
+      python3-ldap \
+      python3-pip \
+      redis \
+      sqlite3 \
+ && apt-get clean \
+ && apt-get autoremove --yes \
+ && rm -rf \
+      /var/lib/apt/lists/* \
+      /etc/nginx/sites-enabled/default \
+ && mkdir -p \
+      /var/log/carbon \
+      /var/log/graphite
+
+COPY conf /
+COPY conf /etc/graphite-statsd/conf/
 
 # copy /opt from build image
 COPY --from=build /opt /opt
 
-# config nginx
-RUN rm /etc/nginx/sites-enabled/default
-ADD conf/etc/nginx/nginx.conf /etc/nginx/nginx.conf
-ADD conf/etc/nginx/sites-enabled/graphite-statsd.conf /etc/nginx/sites-enabled/graphite-statsd.conf
-
-# config redis
-ADD conf/etc/redis/redis.conf /etc/redis/redis.conf
-
-# config collectd
-ADD conf/etc/collectd/collectd.conf /etc/collectd/collectd.conf
-
-# logging support
-RUN mkdir -p /var/log/carbon /var/log/graphite /var/log/nginx /var/log/graphite/
-ADD conf/etc/logrotate.d/graphite-statsd /etc/logrotate.d/graphite-statsd
-
-# daemons
-ADD conf/etc/service/carbon/run /etc/service/carbon/run
-ADD conf/etc/service/carbon-aggregator/run /etc/service/carbon-aggregator/run
-ADD conf/etc/service/graphite/run /etc/service/graphite/run
-ADD conf/etc/service/statsd/run /etc/service/statsd/run
-ADD conf/etc/service/nginx/run /etc/service/nginx/run
-ADD conf/etc/service/redis/run /etc/service/redis/run
-ADD conf/etc/service/collectd/run /etc/service/collectd/run
-RUN chmod 0755 /etc/service/*/run
-
-# default conf setup
-ADD conf /etc/graphite-statsd/conf
-ADD conf/etc/my_init.d/01_conf_init.sh /etc/my_init.d/01_conf_init.sh
-
-# init django admin
-ADD conf/usr/local/bin/django_admin_init.exp /usr/local/bin/django_admin_init.exp
-ADD conf/usr/local/bin/manage.sh /usr/local/bin/manage.sh
-RUN chmod +x /usr/local/bin/manage.sh && /usr/local/bin/django_admin_init.exp
+RUN /usr/local/bin/django_admin_init.exp
```

Comment:
Ahem... but where all stuff went? Nginx/redis/collectd/logging/services?

Reply:
Ah, disregard that, I read your commits. :)
But what about /var/log/carbon /var/log/graphite /var/log/nginx /var/log/graphite/ then?

Reply:
And rm /etc/nginx/sites-enabled/default